README.md

```diff
@@ -197,6 +197,28 @@ Set how many versions of each rpm to keep.
 
 - *Default*: `2`
 
+### `use_lockfile`
+Prevents corruption of the repodata, when multiple createrepo processes
+start building repodata at the same time. (eg in combination with incrond)
+
+- *Default*: `false`
+
+### `lockfile`
+full path/name of the lockfile
+
+- *Default*: `/tmp/createrepo-update-${name}.lock`
+
+### `use_filter`
+For use with `incrond`. Check whether the update script was called with an `.rpm` file as argument,
```

Comment:
If this feature makes it's way into the module I would like the seemingly explicit reference to `incrond` to either be removed or mentioned as an "example use case". There are probably other use cases for this sleeping functionality, it should also probably be renamed to something more clear, although I'm drawing a blank at the moment. "stabilize"? "wait_for_arrival_of_new_packages"? :)

Reply:
See one of the comments below. I'll remove this feature.